controller_manager:
  ros__parameters:
    update_rate: 100

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

    biped_controller:
      type: joint_trajectory_controller/JointTrajectoryController


biped_controller:
  ros__parameters:
    joints:
      - L_hip_roll
      - L_hip_pitch
      - L_knee
      - L_ankle_roll
      - L_ankle_pitch
      - R_hip_roll
      - R_hip_pitch
      - R_knee
      - R_ankle_roll
      - R_ankle_pitch

    command_interfaces:
      - position

    state_interfaces:
      - position
      - velocity

    interpolation_method: splines
    allow_nonzero_velocity_at_trajectory_end: false

    constraints:
      stopped_velocity_tolerance: 0.01
      goal_time: 0.5

